src/components/ec-metroline-card/ec-metroline-card.story.js

```diff
@@ -15,6 +15,9 @@ export const basic = (args, { argTypes }) => ({
       <ec-metroline-card
         class="tw-mb-24"
         :is-collapsed="isCollapsed"
+        :is-first="isFirst"
+        :is-last="isLast"
+        :is-stand-alone="isStandAlone"
```

Comment:
I'm a bit confused about the `isStandAlone` part, wouldn't it be redundant with first and last true?

Also, it is not being tested in the storybook (all false).

Reply:
you're right, the `isStandAlone` prop is actually not necessary for first and last cases, but since we still need it for `Metroline Cards` with more than two items, for the rest of items, I preferred to leave it for homogeneity. Nevertheless, I will improve it so that, if we are in the case of `first` or `last` item, there is no need to also check `isStandAlone`.

The `isStandAlone` prop is used by `Metroline Card` to know if it is being used inside a `Metroline Cards`. Its value by default is `true` (and it is like is displayed in its section in storybook). If it is used within a `Metroline Cards` component the `isStandAlone` prop must be set to `false` (and it is like is displayed in the `Metroline Cards` section in storybook).